docs/guides/edgemesh_test_env_guide.md

```diff
@@ -183,11 +175,11 @@ spec:
         - -c
         - "9292"
         name: init1
-        image: docker.io/ytsobd/edgemesh_init:v1.0
+        image: docker.io/arcanique/edgemesh_init:v1.0
```

Comment:
The image is pushed to kubeedge dockerhub repo, can we use that link ?

Reply:
ok， I modified. please take a look again